src/components/Dashboard/TodoList.vue

```diff
@@ -892,16 +897,17 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
         affiliationInvitationDetails: {
           id: affiliationInvitation.id,
           fromOrgName: affiliationInvitation.fromOrg.name,
-          additionalMessage: affiliationInvitation.additionalMessage
+          additionalMessage: affiliationInvitation.additionalMessage ? affiliationInvitation.additionalMessage : ''
```

Comment:
A simpler way to do this is
`additionalMessage: affiliationInvitation.additionalMessage || ''`